homeassistant/components/devolo_home_network/__init__.py

```diff
@@ -98,7 +98,12 @@ async def async_update_firmware_available() -> UpdateFirmwareCheck:
         try:
             return await device.device.async_check_firmware_available()
         except DeviceUnavailable as err:
-            raise UpdateFailed(err) from err
+            raise UpdateFailed(
+                err,
+                translation_domain=DOMAIN,
+                translation_key="update_failed",
```

Comment:
This translation key doesn't seem to exists

```suggestion
                translation_key="update_failed",
```

Reply:
My bet, I thought it's some default translation key.